### ISSUE TITLE
Disable macros for the menu toolbar

### ISSUE THREAD

**Issue by VRViz** [texstudio-org/texstudio#1697]
### Environment
- TeXstudio: 3.1.2
- Qt: 5.15.2, compiled with Qt 5.15.2 R
- OS: Windows 10
- TeX distribution: texlive

### Description
Hello, please excuse any misplacement (false tags/wrong place to ask) I might have, as I am fairly new here.
Nevertheless I would like to ask for advice regarding the macros for the menu toolbar in texstudio, such as alt+f, alf+e, alt+i, alt+t and so on.
Is there a way as to disable/delete those macros? They take up a fairly big amout of space if you want your personal macros limited like me on the alt key, which makes sorting your makros in incremental order like (alt+)q,w,e,r,t,z impossible. I can´t seem to find those macros in the "Shortcuts" section in the "Configure TeXstudio" menu which lets me think they are stored somewhere else. 
Any help would be greatly appreciated, thanks for taking the time.

**Comment by sunderme** [comment]
We are talking about Menu-Shortcuts ?
<img width="457" alt="grafik" src="https://user-images.githubusercontent.com/14033169/125205140-20918b80-e281-11eb-90ec-4c0784a2f0d7.png">
They are normally demanded by GUI style. I am not aware how to disable them, though it should be possible. (though not by the user)

**Comment by VRViz** [comment]
Yes @sunderme, exactly those. I suspect it might not be possible as these shortcuts displays some basic animations and thus being too integrated into the system as to disable them. However I figured it doesn´t hurt to ask, as they might very well be someone else more experienced then me who was just as annoyed and who found a way to delete some lines in a specific file somewhere. Still hoping for that.
